src/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { includes } from 'ramda';
+
+import { BreadcrumbItemProps } from './Breadcrumbs.types';
+import { getColor, getFontWeight } from '../../utils/helpers';
+import { LinkProps } from '../typography/Link/Link.types';
+import { Link } from '../typography/Link';
+
+const BreadcrumbLink = styled(Link).withConfig<
+  LinkProps & { isSelected: boolean }
+>({
+  shouldForwardProp: (property) => !includes(property, ['$isSelected']),
+})`
+  font-weight: ${getFontWeight('regular')};
+    &,
+    &:not([href]):not([tabindex]) {
+      color: ${({ $isSelected }) =>
+        $isSelected ? getColor('neutral.700') : getColor('neutral.900')}; };
+    }
+
+    &:hover,
+    &:not([href]):not([tabindex]):hover {
+      color: ${getColor('primary.600')};
+    }
+
+    &:active,
+    &:not([href]):not([tabindex]):active {
+      color: ${getColor('primary.700')};
+    }
+`;
+
+const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({
+  children,
+  isSelected = false,
+  ...props
+}) => {
+  return (
+    <li>
+      <BreadcrumbLink
```

Comment:
We should add handling for the last/current item. Right now if you provide an empty `href` or `to` property, the current item still has on hover effect and is clickable. The last element should be non-interactive. I think we should be safe to check if `href` or `to` are undefined and if yes we should render just `Text` component

Reply:
You're right. The last element does not have any hover state.

Reply:
So by definition, the last item only should be marked as "last item" but we would change to "without href and without to".  That's fine. 
I improved a little bit that part.  Just check it out again please